wagtailmodelchoosers/wagtail_hooks.py

```diff
@@ -17,10 +17,14 @@ def wagtailmodelchoosers_admin_css():
 
 @hooks.register('insert_editor_js')
 def wagtailmodelchoosers_admin_js():
-    return format_html(
```

Comment:
I tried it on my machine and did not encounter the same error (although I'm using the test server, not the client's project), however it wasn't working either so I had to change `(static(filename) for filename in js_files)` to `((static(filename),) for filename in js_files)`.